RHCLOUD-31230 Adding virtual-assistant's body (#4)
patternfly/virtual-assistant: packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/VirtualAssistant/VirtualAssistant.md

Reviewer: Can you please add 'VirtualAssistantActions' to propComponents?

Author: Tried, but it doesn't look good (it's not adding all the attributes), it's probably going to cause more confusion. 
I think the reason is because I'm using the same props as Button.

I'll add it if you think that's fine.

```
export type VirtualAssistantActionProps = ButtonProps;
```

![image](https://github.com/patternfly/virtual-assistant/assets/3845764/e55fc46f-11c6-4167-ad3f-2210e0476fcd)

Reviewer: Hmm, yeah, it looks weird, let's not add it for now. I'll reach out to PF on how we could display such types properly. Thank you 🙂

---

import VirtualAssistant from '@patternfly/virtual-assistant/dist/dynamic/VirtualAssistant';

> import VirtualAssistantAction from '@patternfly/virtual-assistant/dist/cjs/VirtualAssistantAction';

Reviewer: ```suggestion
import VirtualAssistantAction from '@patternfly/virtual-assistant/dist/dynamic/VirtualAssistantAction';
```

import { AngleDownIcon } from '@patternfly/react-icons';

> The **virtual assistant** description

Reviewer: ```suggestion
The **virtual assistant** component renders body of the virtual assistant window.
```

### Basic example

> Blank basic example of virtual assistant

Reviewer: ```suggestion
A blank example of the virtual assistant body.
```

```js file="./VirtualAssistantExample.tsx"

```

> ### Setting a different title and placeholder for the input

Reviewer: ```suggestion
### Customizing input title and placeholder
```

You can configure a different title and placeholder input value

```js file="./VirtualAssistantCustomText.tsx"

```

### Listening to messages

> You can listen to the send button using `onSendMessage`.

Reviewer: ```suggestion
The `onSendMessage` property can be used for listening to the send button click.
```

```js file="./VirtualAssistantMessages.tsx"

```

### Disabling send button

> Disabling the send button prevents from being pressed.

Reviewer: ```suggestion
Disabling the send button using `isSendButtonDisabled` prevents it from being clicked.
```

```js file="./VirtualAssistantDisableOnEmptyText.tsx"

```

### Using custom actions

> You can add custom actions to your assistant

Reviewer: ```suggestion
Custom actions can be added to the assistant body using the `actions` property.
```
